pom.xml

```diff
@@ -1760,6 +1767,19 @@
             </execution>
           </executions>
         </plugin>
+
+        <plugin>
+          <groupId>io.github.zlika</groupId>
+          <artifactId>reproducible-build-maven-plugin</artifactId>
+          <version>${reproducible-build-maven-plugin.version}</version>
+          <executions>
+            <execution>
+              <goals>
+                <goal>strip-jar</goal>
```

Comment:
Can you describe the differences to the jar? I tried a little of this a while ago and was not sure about what was OK to remove.